### ISSUE TITLE
Cannot run example projects

### ISSUE THREAD

**Issue by dustinjoe** [deepforge-dev/deepforge#1755]
Hi.
After I set up the DeepForge using the docker-compose method, I tried to import projects from that example repo.
Tried loading project webgmex files from MNIST, Cifar10 and execute pipelines embedded.

But running into errors( cifar10's as follows ) under Sever Execution+Webgme Blob Storage:

170500096/170498071 [==============================] - 801s 5us/step
x_train shape: (50000, 32, 32, 3)
50000 train samples
10000 test samples
############### "GetCifar10Data" Operation Complete! ###############

Using TensorFlow backend.
/root/miniconda3/envs/deepforge/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/root/miniconda3/envs/deepforge/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/root/miniconda3/envs/deepforge/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/root/miniconda3/envs/deepforge/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/root/miniconda3/envs/deepforge/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/root/miniconda3/envs/deepforge/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
/root/miniconda3/envs/deepforge/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/root/miniconda3/envs/deepforge/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/root/miniconda3/envs/deepforge/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/root/miniconda3/envs/deepforge/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/root/miniconda3/envs/deepforge/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/root/miniconda3/envs/deepforge/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])

Also here are some outputs from DEBUG console:
server_1  |   np_resource = np.dtype([("resource", np.ubyte, 1)])
server_1  |  (/K/B/W)
server_1  | 2020-07-12T19:25:02.735Z - info: [gme:server:worker:simpleworker:pid_33:WorkerFunctions:PluginManagerBase:plugin:ExecutePipeline] Save finished w/ status: SYNCED
server_1  | 2020-07-12T19:25:02.790Z - error: [gme:server:worker:simpleworker:pid_33:WorkerFunctions] plugin [ExecutePipeline] failed with error 
server_1  | {
server_1  |   message: 'Operation "GetCifar10Data" failed!',
server_1  |   stack: 'Error: Operation "GetCifar10Data" failed!\n' +
server_1  |     '    at PluginNodeManager.PluginManagerBase.runPluginMain (/deepforge/node_modules/webgme/node_modules/webgme-engine/src/plugin/managerbase.js:341:48)\n' +
server_1  |     '    at runMicrotasks (<anonymous>)\n' +
server_1  |     '    at processTicksAndRejections (internal/process/task_queues.js:97:5)'
server_1  | }
server_1  | 2020-07-12T19:37:25.664Z - info: [gme:server:standalone:JobLogsAPI] Requested logs for /K/B/W in guest+cifar10
server_1  | 2020-07-12T19:37:25.664Z - info: [gme:server:standalone:JobLogsAPI:JobLogManager] Getting log content from /data/blob/log-storage/guest+cifar10/master/_K_B_W.txt
server_1  | 2020-07-12T19:40:23.150Z - error: [gme:server:standalone:middleware:BlobServer:BlobBackend] readStream on close
server_1  | 2020-07-12T19:40:27.526Z - error: [gme:server:standalone:middleware:BlobServer:BlobBackend] readStream on close
server_1  | 2020-07-12T19:40:27.527Z - error: [gme:server:standalone:middleware:BlobServer:BlobBackend] readStream on close

I was asked to click the 'applying update' after importing the file. But I tried click or not click update option, both seem cannot run correctly. Any suggestions? Thanks.

**Comment by brollb** [comment]
Thanks for reporting this, @dustinjoe!

tl;dr - Remove "DEEPFORGE_HOST" from the docker-compose.yml file and then it should work. This will be removed by default following https://github.com/deepforge-dev/deepforge/pull/1756. It is worth mentioning that `DEEPFORGE_HOST` is required for  using SciServer compute with WebGME Blob Storage.

It looks like I forgot to mention in the quickstart that the "DEEPFORGE_HOST" environment variable needs to be configured (or just removed if you are running locally). I had initially added the environment variable there to make it explicit in case anyone wanted to use the file for spinning up a public deployment. However, I think it is actually simpler to just strip down the docker-compose file a little more and then just refer [here](https://github.com/deepforge-dev/deepforge/blob/master/.deployment/docker-compose-overrides.yml) for configuration information for a public deployment.

**Comment by dustinjoe** [comment]
Hi, Brian, thanks for fast reply!
I am running it locally so I just removed it as you suggested and get it running now.
This is a great tool! Thank you for your work!

**Comment by dustinjoe** [comment]
By the way, would it be helpful to add sklearn package to the docker image?
I feel that that would make up a more complete data analysis toolchain?
Thanks.

**Comment by brollb** [comment]
Yeah, that's a good idea. I just made an issue to track the discussion!
